test/build.info

```diff
@@ -436,10 +436,8 @@ INCLUDE_MAIN___test_libtestutil_OLB = /INCLUDE=main
   # available through the shared library (at least on Linux, Solaris, Windows
   # and VMS, where the exported symbols are those listed in util/*.num), these
   # programs are forcibly linked with the static libraries, where all symbols
-  # are always available.  This excludes linking these programs natively on
-  # Windows when building shared libraries, since the static libraries share
-  # names with the DLL import libraries.
-  IF[{- $disabled{shared} || $target{build_scheme}->[1] ne 'windows' -}]
+  # are always available.
+  IF[1]
```

Comment:
Do you intend to retain the possibily to revert the change? Or why didn't you remove the if block and reindent?

Reply:
I chose for minimal change here.  I think there is value in grouping internals testers (you may have noticed the recipes with "internal" in their names too).

I'll leave a more thorough cleanup for another PR (and possibly someone else :wink:)